source/val/validate_memory.cpp

```diff
@@ -323,6 +324,40 @@ spv_result_t ValidateVariable(ValidationState_t& _, const Instruction* inst) {
              << "Classes: Workgroup, CrossWorkgroup, Private, and Function";
     }
   }
+
+  // Vulkan spec 14.5.2
+  if (spvIsVulkanEnv(_.context()->target_env) &&
+      storage_class == SpvStorageClassUniformConstant) {
+    const auto pointer_type_index = 2;
+    const auto pointer_type_id =
+        result_type->GetOperandAs<uint32_t>(pointer_type_index);
```

Comment:
I'd just inline the 2 here (I know we do this in other places, but I don't think it adds much)

Reply:
Done